.github/workflows/pr-close.yml

```
@@ -0,0 +1,29 @@
+name: Cleanup PR preview
+
+# Default to bash
+defaults:
+    run:
+        shell: bash
+
+on:
+    pull_request:
+        types: [closed]
+
+jobs:
+    cleanup_preview:
+        name: Cleanup Preview
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+            - uses: actions/setup-node@v3
```

Please upgrade these to
```suggestion
            - uses: actions/checkout@v4
            - uses: actions/setup-node@v4
```

---

There is a push from GitHub to deprecate node16 and use node20 instead.

Most of the popular actions released a new version to update the default node runtime to 20. We should start using those.

Context: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/
Warning:
```
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: actions/checkout@v3, actions/setup-node@v3.
 ```